services/graphql-server/src/graphql/definitions/platform/content/types/index.js

```diff
@@ -22,7 +22,9 @@ const product = require('./product');
 const promotion = require('./promotion');
 const review = require('./review');
 const sponsored = require('./sponsored');
+const supplier = require('./supplier');
```

Comment:
These new content types need to be added to the https://github.com/base-cms/base-cms/blob/master/packages/utils/src/content-types.js file